recipes/spam/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set name = "spam" %}
+{% set version = "0.6.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gricad-gitlab.univ-grenoble-alpes.fr/ttk/spam/-/archive/62a7c325dbea0d90314ba19a0ac76a61a23691f4/spam-62a7c325dbea0d90314ba19a0ac76a61a23691f4.tar
+  sha256: bc1c898392eac220784ae534550e2b3b0c071053f9aca870264cc79718b8886e
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python       # [unix]
+    - python =3.7  # [win]
+    - numpy
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - {{ compiler('cxx') }}
+    - pybind11
+    - eigen
+    - cgal-cpp
+    - gmp                                    # [unix]
+    - m2w64-gmp                              # [win]
+    - pyamg
+  host:
+    - python       # [unix]
+    - python =3.7  # [win]
+    - pip
+    - numpy
+    - pybind11
+    - eigen
+    - cgal-cpp
+    - gmp                                    # [unix]
+    - m2w64-gmp                              # [win]
+  run:
+    - python       # [unix]
+    - python =3.7  # [win]
+    - matplotlib-base
+    - gmp                                    # [unix]
+    - m2w64-gmp                              # [win]
+    - mpir                                   # [win]
+    - r-base
+    - numpy
+    - scipy
+    - scikit-image
+    - tifffile
+    - matplotlib-base
+    - SimpleITK
+    - progressbar
+    - python-decouple
+    - gmsh
+    - meshio =4.3.13
+    - h5py
+
+test:
+  imports:
+    - spam.DIC
+    - spam.helpers
+  requires:
+    - python
```

Comment:
you can omit anything in `run` here

Reply:
We're not sure to have understood. Do you mean:
1.  we should remove everything in `requirements` -> `run`
2. or in `test` -> `requires` (as the comment is made in this section)
3. or everything that is in `requirements` -> `run` do not need to be repeated in `test` -> `requires`

We did nothing for the moment.

Reply:
@Kivou-2000607 the created package and all it's run dependencies will be installed in the test environemnt. it's much better to not repeat the dependenceis to make sure you haven't forgotten any in `run`.